platforms/BUILD.bazel

```diff
@@ -101,3 +101,11 @@ platform(
         "//cc/constraints:yocto_generic_toolchain",
     ],
 )
+
+platform(
+    name = "aarch64_linux",
```

Comment:
Let's name this platform `bullseye_aarch64` (or maybe `aarch64_bullseye`, what do you think?).

We should add an extra constraint value to distinguish it.

I'm thinking either introduce a new `constraint_setting` for `glibc` version, or for `debian`, and then the `constraint_values` could either be `glibc_2_2x` or `bullseye` respectively.

Reply:
Added glibc `constraint_value` for libc `constraint_setting`.